api/notd/api/api_v1.py

```diff
@@ -76,6 +79,12 @@ async def receive_new_blocks_deferred():
         await notdManager.reprocess_old_blocks_deferred()
         return ReceiveNewBlocksDeferredResponse()
 
+    #NOTE CHANGE RESPONSE TO MATCH FUNCTION
```

Comment:
is this sill valid? if it is, the format for notes should be:

```
# NOTE(<github_username>): Something to note
```

(see my NOTEs or TODOs for examples)